cmd/rollapp/init/utils.go

```diff
@@ -91,6 +91,23 @@ func runInit(
 			return fmt.Errorf("failed to get Avail account address: %w", err)
 		}
 
+		// Append DA account address if available
+		if daAddress != nil {
+			addresses = append(addresses, keys.KeyInfo{
+				Name:    damanager.GetKeyName(),
+				Address: daAddress.Address,
+			})
+		}
```

Comment:
is this intentionally here? it's in an avail block, in general - it's easier/cleaner to handle it outside the switch statement, isn't it?

Reply:
I think only Avail and WeaveVm use this block, Celes doesn't. Putting it outside the switch statement might be inappropriate.